tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/binaryoperations/NegativeTypeTestExprTest.java

```diff
@@ -0,0 +1,747 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Shall we change this in all new files?

Reply:
okay. Will change